bot/src/modules/birthday/commands/birthday/toggle.ts

```diff
@@ -45,16 +52,21 @@ export default SubCommand(
       return { embeds: [failEmbed] };
     }
 
-    await state.db.userGuildConfig.upsert({
+    const birthday = await state.db.userGuildConfig.upsert({
       where: { userId_guildId: { guildId: msg.guild.id, userId: msg.user.id } },
-      create: { userId: msg.user.id, guildId: msg.guild.id, birthdayEnabled: true },
-      update: { birthdayEnabled },
+      create: { userId: msg.user.id, guildId: msg.guild.id, birthdayEnabled: true, birthdayAnnounceEnabled },
+      update: { birthdayEnabled, birthdayAnnounceEnabled },
     });
 
-    let description = `Your birthday is now ${birthdayEnabled ? "visible" : "hidden"} in this server`;
+    let description = `Your birthday is now **${birthday.birthdayEnabled ? "visible" : "hidden"}** in this server`;
+
+    if (!birthday.birthdayAnnounceEnabled || !birthday.birthdayEnabled)
+      description += `, ${birthday.birthdayEnabled ? "but" : "and"} will **not** be announced"`;
```

Comment:
This implies that `birthdayEnabled = false` should disable the announcements, but [birthday/cron/loop.ts#R33-R37](https://github.com/Owl-barn/bot/pull/39/files#diff-4f8a350396d16013cbee80c10afc0867d933cff3d6b0c9b29b813352086b3b4aR33-R37) seems like it only checks if announce is enabled